common/web/input-processor/test.sh

```diff
@@ -73,13 +73,13 @@ test-headless ( ) {
 if [ $FETCH_DEPS = true ]; then
   # Next, build the lm-worker in its proper, wrapped form
   pushd "$KEYMAN_ROOT/common/web/lm-worker"
-  ./build.sh
+  ./build.sh --report-scope
```

Comment:
I think `--report-scope` should be always on. It's short and clear in the logs. Can you update accordingly? Should simplify this PR too :grin:

Reply:
I did think about that, but figured you'd go the other way with your decision; can do.

That also means there's no need for the new script option to exist (b/c always on), so I'll get to work on removing it.